API/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Base image
+FROM node:16
```

Comment:
We may want to consider a specific version of node so we don't get caught out by minor updates breaking things.

Reply:
Constraining it to 16.16.0